demo/x11/Makefile

```diff
@@ -1,5 +1,6 @@
 # Install
 BIN = zahnrad
+FIX = fix
```

Comment:
I think to better distinguish the two binaries/apps, we should rather use `EDITFIX` instead of just `FIX` (same goes for `fix` -> `editfix`).